src/pages/documentation/resources/acp-end-of-support.md

```diff
@@ -0,0 +1,25 @@
+# ACP SDKs end-of-support
+
+## End of support for ACP iOS SDKs and Wrappers
```

Comment:
Can you also add an info block marked as important at top of the home page that links to this page?